etc/cryptor-mock/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM	node:alpine
+
+WORKDIR /src
+
+RUN npm i express body-parser
+COPY	cryptor-mock.js /src
+
+EXPOSE	26272
+CMD	node /src/cryptor-mock.js
```

Comment:
Do we know for sure there's not a cryptor mock already in existence?  If there isn't, should there be one on docker hub to make it easier for everyone to dev with it in the future?  

I'm not sure which team we'd need to talk to about this.